mlflow/utils/uri.py

```diff
@@ -8,7 +8,7 @@
 
 _INVALID_DB_URI_MSG = "Please refer to https://mlflow.org/docs/latest/tracking.html#storage for " \
                       "format specifications."
-
+_ACLED_ARTIFACT_URI = "dbfs:/databricks/mlflow-tracking/"
```

Comment:
Nit: Can we move this into `is_artifact_acled_uri()`? It doesn't seem to be required by more than one function in this module.

Reply:
Done.